### PR TITLE
Drop unicode literal u prefix

### PR DESCRIPTION
**Pull request by peterjc**
We included these for Python 2 compatibility, but are now redundant.

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Tests/test_PhyloXML.py [hunk 1]
```diff
@@ -305,14 +305,17 @@ def test_Distribution(self):
...
-                    u"ETH Z\xfcrich",
-                    "San Diego"),
-                (47.481277, 35.155904, 47.376334, 32.880933),
-                (8.769303, 136.915863, 8.548108, -117.217543),
-                (472, 10, 452, 104)):
+            (hirschweg, nagoya, eth_zurich, san_diego),
+            (
+                "Hirschweg, Winterthur, Switzerland",
+                "Nagoya, Aichi, Japan",
+                "ETH Z\xfcrich",
```
Comment (mdehoon) [inline Tests/test_PhyloXML.py]: Shouldn't this be Zürich?

File: Tests/test_SearchIO_blast_xml.py [hunk 2]
```diff
@@ -18,10 +18,12 @@
 TEST_DIR = "Blast"
 FMT = "blast-xml"
 
-REFERENCE = (u"Altschul, Stephen F., Thomas L. Madden, Alejandro A. Sch\xe4ffer, "
-             u"Jinghui Zhang, Zheng Zhang, Webb Miller, and David J. Lipman (1997), "
-             u'"Gapped BLAST and PSI-BLAST: a new generation of protein database '
-             u'search programs", Nucleic Acids Res. 25:3389-3402.')
+REFERENCE = (
+    "Altschul, Stephen F., Thomas L. Madden, Alejandro A. Sch\xe4ffer, "
```
Comment (mdehoon) [inline Tests/test_SearchIO_blast_xml.py]: Schäffer?

File: Tests/test_SeqIO_SeqXML.py [hunk 3]
```diff
@@ -48,13 +48,19 @@ def setUp(self):
...
+        )
 
     # TODO - Fix this failure under Windows with Python 3.1 and 3.2
     if not (sys.platform == "win32" and sys.version_info[0] >= 3):
         def test_unicode_characters_desc(self):
             """Test special unicode characters in the description."""
-            self.assertEqual(self.records["rna"][2].description, u"\u00E5\u00C5\u00FC\u00F6\u00D6\u00DF\u00F8\u00E4\u00A2\u00A3$\u20AC\u9999\u80A0")
+            self.assertEqual(
+                self.records["rna"][2].description,
+                "\u00E5\u00C5\u00FC\u00F6\u00D6\u00DF\u00F8\u00E4\u00A2\u00A3$\u20AC\u9999\u80A0"
```
Comment (mdehoon) [inline Tests/test_SeqIO_SeqXML.py]: åÅüöÖßøä¢£$€香肠

File: Tests/test_SearchIO_blast_xml.py (hunk 2 above)
Comment (peterjc) [reply Tests/test_SearchIO_blast_xml.py]: Yes, that's what the ``\xe4`` means.

**Comment by peterjc** [comment]
Do we need to do anything special to declare the Python file's encoding if we start putting unicode characters directly in it?

**Comment by peterjc** [comment]
Looks like we can assume Python will interpret ``*.py`` content as UTF8, and just use unicode directly:

https://docs.python.org/3/howto/unicode.html

**Comment by peterjc** [comment]
Literal unicode used, rebased for a merge conflict.

**Comment by peterjc** [comment]
Thanks both

**Pull request merged**